### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Dqnns**
Removed unnecessary nouns and deleted a few 2 character usernames.

### CODE REVIEW

**Comment by bribes** [comment]
> This branch has conflicts that must be resolved
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

rip

**Comment by bribes** [comment]
Nathaniel is a real name.

File: src/names/names.txt [hunk 1]
```diff
@@ -2031,79 +2020,36 @@ Monique
...
-Nataliya
-Natalja
-Natalka
 Natalya
 Natascha
 Natasha
 Nate
 Nathalie
 Nathan
-Nathaniel
```
Comment (bribes) [inline src/names/names.txt:2090]: Nathaniel is a real name.

**Review by bribes — CHANGES_REQUESTED** [review]
Nathaniel is a real name that is used IRL.

File: src/names/names.txt [hunk 2]
```diff
@@ -2033,79 +2022,36 @@ Monique
...
```
Comment (bribes) [inline src/names/names.txt:2090]: Nathaniel is a real name.

**Comment by Dqnns** [comment]
> 

Great, thanks ^^

**Pull request closed without merging**